R/timeprofile-plotconfiguration.R

```diff
@@ -23,16 +26,18 @@ TimeProfilePlotConfiguration <- R6::R6Class(
                           y2Axis = NULL,
                           y2Scale = NULL,
                           y2Limits = NULL,
+                          lloqDirection = "horizontal",
                           # Smart configuration using metaData
                           data = NULL,
                           metaData = NULL,
                           dataMapping = NULL) {
       super$initialize(
-        ..., 
+        ...,
         data = data,
         metaData = metaData,
-        dataMapping = dataMapping
+        dataMapping = dataMapping,
```

Comment:
This `,` is wrong?

Reply:
I'll remove it

Reply:
Obviously, a test is missing for creating a `TimeProfilePlotConfiguration`, right? Otherwise this would have been caught by a test.

Reply:
No, it does not break the code:
```r
TimeProfilePlotConfiguration$new()
<TimeProfilePlotConfiguration>
  Inherits from: <PlotConfiguration>
  Public:
    background: active binding
    clone: function (deep = FALSE) 
    defaultExpand: FALSE
    defaultSymmetricAxes: FALSE
    defaultXScale: lin
    defaultYScale: lin
    errorbars: active binding
    export: ExportConfiguration, R6
    initialize: function (..., y2label = NULL, y2Axis = NULL, y2Scale = NULL, 
    labels: active binding
    legend: active binding
    lines: active binding
    lloqDirection: horizontal
    points: active binding
    ribbons: active binding
    xAxis: active binding
    y2Axis: active binding
    yAxis: active binding
  Private:
    .background: BackgroundConfiguration, R6
    .errorbars: ThemeAestheticSelections, ThemeAestheticMaps, R6
    .labels: LabelConfiguration, R6
    .legend: LegendConfiguration, R6
    .lines: ThemeAestheticSelections, ThemeAestheticMaps, R6
    .points: ThemeAestheticSelections, ThemeAestheticMaps, R6
    .ribbons: ThemeAestheticSelections, ThemeAestheticMaps, R6
    .xAxis: XAxisConfiguration, AxisConfiguration, R6
    .y2Axis: YAxisConfiguration, AxisConfiguration, R6
    .yAxis: YAxisConfiguration, AxisConfiguration, R6
```

I added a unit test nonetheless.